<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>801</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Calendars:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="8">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QListWidget" name="calendar_list_listWidget"/>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="send_code_pushButton">
       <property name="text">
        <string>Send Code</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4" colspan="4">
      <widget class="QListWidget" name="event_list_listWidget"/>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QLineEdit" name="calendar_id_lineEdit"/>
     </item>
     <item row="5" column="4" colspan="4">
      <widget class="QLineEdit" name="event_id_lineEdit"/>
     </item>
     <item row="6" column="6">
      <widget class="QPushButton" name="event_edit_pushButton">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="7">
      <widget class="QLineEdit" name="send_code_lineEdit"/>
     </item>
     <item row="6" column="7">
      <widget class="QPushButton" name="event_delete_pushButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QPushButton" name="event_create_pushButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="calendar_create_pushButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="calendar_edit_pushButton">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="refresh_pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="calendar_delete_pushButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Events:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="local_calendar_checkBox">
       <property name="text">
        <string>Local Calendar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="login_pushButton">
       <property name="text">
        <string>Login to Google</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="remote_calendar_checkBox">
       <property name="text">
        <string>Google Calendar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <signal>closed()</signal>
 </slots>
</ui>
